build-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e 
+
+# This script is used to build and tag both docker images while still
+# keeping the images in the same Dockerfile. 
+# Doing this ensures that both images will have synced dependencies. 
+
+BASE_TAG="webdev"
+VALIDATE_TAG="${BASE_TAG}-validator"
```

Comment:
Curly braces aren't needed here, but it's not hurting, so it's up to you.

```suggestion
VALIDATE_TAG="$BASE_TAG-validator"
```

Reply:
I've gone ahead and removed this, but generally if I'm adding text to the bash variable, I like to use something to separate the variable name from the additional text.